python/tests/onnx_quantizer_tests/layers_tests/test_e2e.py

```
@@ -0,0 +1,196 @@
+        self._check_validation_dependency(test_case_data)
+        if test_spec.skip_reason:
+            pytest.skip(f"{layer_name}_{test_spec.name}: {test_spec.skip_reason}")
+
+        if layer_name == "Constant":
+            pytest.skip(f"No e2e test for layer {layer_name} yet")
+
+        # Create original model
+        original_model = config.create_test_model(test_spec)
+
+        # Save quantized model to temp location
+        import onnx  # noqa: PLC0415
```

Ruff (0.14.4)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)